src/configModel/configModel.vala

```diff
@@ -207,6 +211,7 @@ namespace SwayNotificationCenter {
 #if WANT_SCRIPTING
     public class Script : NotificationMatching {
         public string ? exec { get; set; default = null; }
+        public string ? run_on { get; set; default = "receive"; }
```

Comment:
I'd prefer the use of an enum. It'll make it easier to keep track of in the future